EventHighway.Core.Tests.Unit/Services/Foundations/EventAddresses/EventAddressServiceTests.Logic.Add.cs

```diff
@@ -47,5 +49,38 @@ await this.eventAddressService.AddEventAddressAsync(
 
             this.storageBrokerMock.VerifyNoOtherCalls();
         }
+
+        [Fact]
```

Comment:
This test should live in it's own partial class. "EventAddressServiceTests.Validations.Add"